File: library/src/main/java/io/github/wax911/library/model/body/GraphContainer.java
package io.github.wax911.library.model.body;

import java.util.List;

import io.github.wax911.library.model.attribute.GraphError;

public class GraphContainer<T> {

    private T data;
    private List<GraphError> errors;

    public T getData() {
        return data;
    }

    public List<GraphError> getErrors() {
        return errors;
    }

    public boolean isEmpty() {
        return data == null;
    }
}

File: library/src/main/kotlin/io/github/wax911/library/model/body/GraphContainer.kt
@file:Suppress("DEPRECATION")

package io.github.wax911.library.model.body

@Deprecated(
    message = "Use co.anitrend.retrofit.graphql.model.body.GraphContainer instead",
    replaceWith = ReplaceWith(
        "GraphContainer",
        "co.anitrend.retrofit.graphql.model.body.GraphContainer"
    ),
    level = DeprecationLevel.WARNING
)
public typealias GraphContainer<T> = co.anitrend.retrofit.graphql.model.body.GraphContainer<T>
